src/coreclr/src/jit/optcse.cpp

```diff
@@ -1878,8 +1878,7 @@ class CSE_Heuristic
 
         bool LiveAcrossCall()
         {
-            //            return (m_CseDsc->csdLiveAcrossCall != 0);
-            return false; // The old behavior for now
+            return (m_CseDsc->csdLiveAcrossCall != 0);
```

Comment:
Why does `x64/x84/arm64` regress from it? Do we need an issue to change how we use it because it looks like current usage only make sense for arm32.

Reply:
> Can you also share code size diff reports?

https://gist.github.com/briansull/8518adc549803963e7f5a41c6d338a64

Reply:
The  tuning changes for LiveAccrossCall are made in the follow on changesets, 
Now when I make the LiveAccrossCall change (line 1901) as the only change it show a large improvement:

Top file improvements (PerfScoreUnits):
    -41194.39 : System.Private.CoreLib.dasm (-1.93% of base)

268 total methods with Perf Score differences (154 improved, 114 regressed), 19997 unchanged.

https://gist.github.com/briansull/558437392f1b92e8359bbca8c8782910